site/app/models/Gradeable.php

```diff
@@ -284,11 +284,9 @@ class Gradeable extends AbstractModel {
 
     protected $been_tagraded = false;
 
-    protected $total_tagrading_non_extra_credit = 0;
-    protected $total_tagrading_extra_credit = 0;
+    protected $total_tagrading_points = 0;
```

Comment:
change to `total_ta_grading_points` to be consistent with `total_peer_grading_points` (and also utilize a magic getter like the `getTotalPeerGradingPoints` method does)

Reply:
the magic getter getTotalTAGradingPoints defaults to return total_t_a_grading_points, so I think we have to stick with a non-magic version if we want to keep the expected naming convention.